### DISCUSSION TITLE
php version 8.1.1 compatibility

### DISCUSSION THREAD

**Discussion by jlalonsoredon** [PHPOffice/PhpSpreadsheet#2511] [Q&A] [answered]
I am trying to install phpspreadsheet with Composer but it gives me the following error:
   The phpoffice/phpspreadsheet package has requirements that are incompatible with your PHP version, PHP extensions, and Composer version:
      - phpoffice/phpspreadsheet 1.21.0 requires ext-dom * but it is not present.
I am working with php version 8.1.1.
Should it be compatible?

**Comment by oleibman** [comment, marked as answer]
The dom extension is enabled by default, but it can be disabled when php is being compiled. See https://www.php.net/manual/en/dom.installation.php for some suggestions if it isn't available.

**Reply by jlalonsoredon** [reply]
I don't understand what relationship it has with the PHP version

**Reply by oleibman** [reply]
If the one version of php was compiled without disabling the extension, and the other version was compiled disabling the extension, it will work on the former and not on the latter.
